README.md

```diff
@@ -112,10 +106,10 @@ dnf install antimicrox
 ```
 trizen -S antimicrox
 ```
-**or**<br/>
+**or**
 
 pre-built version can de downloaded from unofficial repository called [chaotic-aur](https://lonewolf.pedrohlc.com/chaotic-aur/).
-<br/>Append (one of listed mirrors) to `/etc/pacman.conf`:
+Append (one of listed mirrors) to `/etc/pacman.conf`:
```

Comment:
Now it looks like:
```
pre-built version can de downloaded from unofficial repository called [chaotic-aur](https://lonewolf.pedrohlc.com/chaotic-aur/). Append (one of listed mirrors) to `/etc/pacman.conf`:
```
instead of
```
pre-built version can de downloaded from unofficial repository called [chaotic-aur](https://lonewolf.pedrohlc.com/chaotic-aur/).
Append (one of listed mirrors) to `/etc/pacman.conf`:
```

If you want to remove this `<br/>` then add two spaces `  ` at the end of previous line (it will preserve formatting) you could apply it in other places where you remove this `<br/>`.